tools/check-node-fixtures

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import subprocess
+import sys
+from typing import Any, Callable, Dict, Optional
+
+import ujson
+
+TOOLS_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, os.path.dirname(TOOLS_DIR))
+ROOT_DIR = os.path.dirname(TOOLS_DIR)
+
+# check for the venv
+from tools.lib import sanity_check
+
+sanity_check.check_venv(__file__)
+
+USAGE = """
+
+    This program reads in fixture data for our
+    node tests, and then it validates the fixture
+    data with checkers from event_schema.py (which
+    are the same Python functions we use to validate
+    events in test_events.py).
+
+    It currently takes no arguments.
+"""
+
+parser = argparse.ArgumentParser(usage=USAGE)
+parser.parse_args()
+
+# We can eliminate the django dependency in event_schema,
+# but unfortunately it"s coupled to modules like validate.py
+# and topic.py.
+import django
+
+os.environ["DJANGO_SETTINGS_MODULE"] = "zproject.test_settings"
+django.setup()
```

Comment:
Hmm, interesting.  If we can avoid this, we can probably make this check fast enough to run in `test-js-with-node`.

Reply:
Yeah, we can break the **entire** Django dependency with basically two changes:
* move all the non-Django validators to `checkers.py` (and just let `validators.py` import them, so the rest of the codebase is oblivious to `checkers.py`)
* move the three topic constants to `topic_constants.py` (and do a similar thing to make the rest of the codebase happy)

There's also a deeper fix, where I just make event_schema.py use schemas that are actually composed from types, not functions, so I may just wait for that.  For now it's definitely fine, since I'm the primary user of the tooling.

Reply:
I would prefer to move the Django validators to a separate file; the vast majority are non-Django validators.

Reply:
Yeah, when I'm ready to make this change, I'll try to ping you.  It's mostly six/half-a-dozen as how we do it, but there are a couple different approaches with different tradeoffs.

Reply:
(I like the notion of moving the least common cases out of the main module, of course.)